<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../out/build/debug/.qt/rcc/icon.qrc">
    <normaloff>:/resources/icon/logo.ico</normaloff>:/resources/icon/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background: #1A1A1A;
color: white;
}</string>
  </property>
  <widget class="QLabel" name="lbSettings">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: #363636;
font: 700 12pt &quot;Arial&quot;;
border-radius: 10px
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbPin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: #363636;
font: 700 12pt &quot;Arial&quot;;
border-radius: 10px
}</string>
   </property>
   <property name="text">
    <string>Pin window to:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cbPinCorner">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
background: #363636;
font: 700 12pt &quot;Arial&quot;;
border-radius: 10px
}
QComboBox:Hover{
background: #696969;
font: 700 12pt &quot;Arial&quot;;
border-radius: 10px
}</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Topleft</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbOpacity">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: #363636;
font: 700 12pt &quot;Arial&quot;;
border-radius: 10px
}</string>
   </property>
   <property name="text">
    <string>Opacity:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetDefault">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: #363636;
	font: 700 12pt &quot;Arial&quot;;
	border-radius: 10px
}
QPushButton:Hover
{
	background: #696969;
	font: 700 12pt &quot;Arial&quot;;
	border-radius: 10px
}
QPushButton:pressed
{
	background: #363636;
	font: 700 12pt &quot;Arial&quot;;
	border-radius: 10px
}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>300</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: #363636;
	font: 700 12pt &quot;Arial&quot;;
	border-radius: 10px
}
QPushButton:Hover
{
	background: #696969;
	font: 700 12pt &quot;Arial&quot;;
	border-radius: 10px
}
QPushButton:pressed
{
	background: #363636;
	font: 700 12pt &quot;Arial&quot;;
	border-radius: 10px
}</string>
   </property>
   <property name="text">
    <string>Save Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHide_Show">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: #363636;
font: 700 12pt &quot;Arial&quot;;
border-radius: 10px
}</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Hide list when playing</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxHide">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
background: #363636;
font: 700 12pt &quot;Arial&quot;;
border-radius: 10px;
padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbDefault">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: #363636;
	font: 700 12pt &quot;Arial&quot;;
	border-radius: 10px
}</string>
   </property>
   <property name="text">
    <string>Reset to default</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSlider" name="sliderOpacity">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Opacity: 0 (Invisible) â†’ 100 (Visible)</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider{
background: #363636;
border-radius: 10px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../out/build/debug/.qt/rcc/icon.qrc"/>
 </resources>
 <connections/>
</ui>
